ddl/ddl_api.go

```diff
@@ -551,6 +555,24 @@ func checkPriKeyConstraint(col *table.Column, hasDefaultValue, hasNullFlag bool,
 	return nil
 }
 
+func checkColumnValueConstraint(col *table.Column) error {
+	if col.Tp == mysql.TypeEnum || col.Tp == mysql.TypeSet {
+		valueMap := make(map[string]string, len(col.Elems))
```

Comment:
use util.StringSet to replace `map[string]string`

Reply:
I need string value in map[string]string to generated error msg.